src/Stryker.Core/Stryker.Core/Initialisation/InitialisationProcess.cs

```diff
@@ -44,7 +44,8 @@ public MutationTestInput Initialize(StrykerOptions options)
             var projectInfo = _inputFileResolver.ResolveInput(options);
 
             // initial build
-            _initialBuildProcess.InitialBuild(projectInfo.FullFramework, options.BasePath, options.SolutionPath, Path.GetFileName(projectInfo.TestProjectAnalyzerResult.ProjectFilePath));
+            _initialBuildProcess.InitialBuild(projectInfo.FullFramework, Path.GetDirectoryName(projectInfo.TestProjectAnalyzerResult.ProjectFilePath), 
```

Comment:
I would prefer the callee becomes responsible for correctly handling getting the directory name from the path since we are dealing with strings and have no mechanism to validate the string input so cannot force the caller to make the right decision.